twake/frontend/src/app/scenes/Client/ChannelsBar/ChannelWorkspaceEditor.tsx

```diff
@@ -59,7 +59,19 @@ const ChannelWorkspaceEditor: FC<Props> = ({ title, channel }) => {
             float: 'right',
           }}
           disabled={!disabled}
-          onClick={() => upsertChannel()}
+          onClick={() => {
+            //upsertChannel()
+            return ModalManager.open(
+              <ChannelMembersEditor
```

Comment:
This is called only on creation of channel (not on edition), we must detect the current state (creation or update) and act accordingly.
I add a task for this in the current sprint :)

Reply:
Here it is: #555